tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -18,6 +18,27 @@
   <mat-icon svgIcon="notifications_none_24px"></mat-icon>
 </button>
 
-<mat-menu #menu="matMenu">
-  <div mat-menu-item>notification 1</div>
+<mat-menu #menu="matMenu" class="notification-menu">
+  <div>
+    <div *ngFor="let notification of notifications" class="menu-item">
+      <div>
+        <mat-icon
+          *ngIf="notification.category"
+          svgIcon="{{ notification.icon }}"
+          class="category-icon"
+        ></mat-icon>
+        <ng-container [ngSwitch]="notification.category">
+          <span *ngSwitchCase="CategoryEnum.WHATS_NEW">What's new</span>
+        </ng-container>
+        <span *ngIf="notification.category">・</span>
+        <span>{{ notification.dateInMs | date:'medium'}}</span>
```

Comment:
If you are inclined, put `[title]="notification.dateInMs | date:'long'"`

Reply:
changed to `full` based on offline sync